### PR TITLE
[CMake] Add flag to disable fuzztesting

### PR DESCRIPTION
**Pull request by RossBrunton**
Default is enabled; this should not be a visible change without changing your cmake configuration.

### CODE REVIEW

**Review by kbenzie — APPROVED** [review]

**Comment by pbalcer** [comment]
Why?

Also, please set this in workflow jobs (there are two - nightly and a normal one).

**Comment by RossBrunton** [comment]
> Why?
> 
> Also, please set this in workflow jobs (there are two - nightly and a normal one).

The compiler I use to build LLVM is a clang without the fuzzing headers available, so I constantly have to comment out the include. Figured I'd just quickly add a variable and push it up in case it was useful to anyone else.

Given that it's enabled by default, does it need to be added to all the cmake invocations?

File: test/CMakeLists.txt [hunk 1]
```diff
@@ -12,6 +12,7 @@ FetchContent_Declare(
 
 set(UR_TEST_DEVICES_COUNT 1 CACHE STRING "Count of devices on which conformance and adapters tests will be run")
 set(UR_TEST_PLATFORMS_COUNT 1 CACHE STRING "Count of platforms on which conformance and adapters tests will be run")
+set(UR_TEST_FUZZTESTS ON CACHE STRING "Run fuzz tests if using clang and UR_DPCXX is specified")
```
Comment (lukaszstolarczuk) [inline test/CMakeLists.txt:15]: I think it should be `BOOL` not `STRING`

**Comment by pbalcer** [comment]
> > Why?
> > Also, please set this in workflow jobs (there are two - nightly and a normal one).
> 
> The compiler I use to build LLVM is a clang without the fuzzing headers available, so I constantly have to comment out the include. Figured I'd just quickly add a variable and push it up in case it was useful to anyone else.
> 
> Given that it's enabled by default, does it need to be added to all the cmake invocations?

Nvm, I thought you disabled it by default. lgtm.

**Pull request merged**